### ISSUE TITLE
Ability to copy "loading page" URL

### ISSUE THREAD

**Issue by LeOSW42** [browsh-org/browsh#302]
Hello there,

I'm using browsh to download files from Wetransfer and I am missing the ability to either:
 * Copy the "Download" link location
 * Copy the "Loading page" URL once the download button is pressed
 * Store a downloaded file.

Léo

**Comment by tombh** [comment]
Yeah, totally agree this is needed. This is the kind of thing that actually makes Browsh useful rather than just a gimmick.
